ops-bedrock/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ volumes:
 services:
   da:
     platform: linux/x86_64
-    image: "ghcr.io/rollkit/local-celestia-devnet:v0.11.0"
+    image: "ghcr.io/rollkit/local-celestia-devnet:v0.12.1"
```

Comment:
We should `docker-compose-testnet.yml` while we are at it.

Reply:
🫡 https://github.com/celestiaorg/optimism/commit/ca7ed14a4a1593e2109960a4b3d07e5a208e0a6b

Reply:
utACK though